### PR TITLE
Add more briefcases to the loadout and rework some icons

### PR DESCRIPTION
**Pull request by TheGoret**
Happy_Fox made sprites for briefcases and I decided to add them. He also reworked some basic hand-sprites, such as the weird outline on the brown one.

Thanks to Happy_Fox for the spriting, and Mwahahahaha for the code from a closed PR.

### CODE REVIEW

**Comment by TheGoret** [comment]
!review

**Comment by TheGoret** [comment]
> 
> 
> The icon_states and item_states of the dmi dont line up with the ones used in the code.
> Are you certain that they work?

Good catch on that one, I forgot to rename the last briefcase. The others are fine.

**Comment by Arrow768** [comment]
Are you certain that it's fine now?
Because thats the name of the sprite in the storage.dmi: `briefcase_corporate`
And those are the names in the lefthand.dmi / righthand.dmi: `briefcase_corpnt` `briefcase_nt`

The item uses `briefcase_corpnt` so something does not line up (and you added unused icons which should be removed or used)

**Comment by TheGoret** [comment]
The briefcase_nt one shouldn't be here, I decided to keep it for later use for another PR involving other contractors. Thank you, it SHOULD be all good now.

File: code/game/objects/items/weapons/storage/briefcase.dm [hunk 1]
```diff
@@ -11,4 +11,22 @@
...
 	use_sound = 'sound/items/storage/briefcase.ogg'
-	drop_sound = 'sound/items/drop/backpack.ogg'
\ No newline at end of file
+	drop_sound = 'sound/items/drop/backpack.ogg'
+
+/obj/item/storage/briefcase/black
+	name = "black briefcase"
+	icon_state = "briefcase_black"
+	item_state = "briefcase_black"
+	desc = "Named for lawyers carrying briefs in a case. It's about time they had something named for them, isn't it?"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/storage/briefcase.dm]:
```suggestion
	desc = "Named after lawyers carrying briefs in a case. It's about time they had some representation, isn't it?"
```

**Pull request merged**